lib/mayaUsd/ufe/UsdTRSUndoableCommandBase.cpp

```diff
@@ -0,0 +1,112 @@
+//
```

Comment:
This is the new base class for shared code between translate, rotate, and scale.